### PR TITLE
Reduce yaml files for lvm+cancel_existing_cryptlvm

### PR DESCRIPTION
**Pull request by okynos**
We have removed the majority of schedule files, PowePC maintained for compatibility.

- Related ticket: https://progress.opensuse.org/issues/163247
- MR: https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/307
- Needles: N/A
- Verification run:
  - Flavor Online: https://openqa.suse.de/tests/overview?distri=sle&version=15-SP6&build=93.5&groupid=583
  - lvm+cancel_existing_cryptlvm:
    - x86_64: https://openqa.suse.de/tests/15168050 (Updated-13/08/2024)
    - aarch64: https://openqa.suse.de/tests/15168032 (Updated-13/08/2024)
    - ppc64le: https://openqa.suse.de/tests/15168149 (Updated-13/08/2024)

### CODE REVIEW

File: schedule/yast/lvm/lvm+cancel_existing_cryptlvm_pvm.yaml [hunk 1]
```diff
@@ -3,9 +3,6 @@ description: >
   Test suite cancels encrypted partitions activation and performs installation
```
Comment (jknphy) [inline schedule/yast/lvm/lvm+cancel_existing_cryptlvm_pvm.yaml:3]: we need to unify this file too, otherwise we cannot create new one in test_cases.

**Comment by jknphy** [comment]
Ideally to make easier the review and have less verbose (less copy/paste in 1st comment) you could just have on link that point to your verification runs. openqa-clone-custom-git-refspec create special branch where you can click see all related with that git branch, so you don't mix all of them. It is not much practical for this kind of verification to point to your job group, in no time will be more mess in it.

File: schedule/yast/lvm/lvm+cancel_existing_cryptlvm_pvm.yaml (hunk 1 above)
Comment (okynos) [reply schedule/yast/lvm/lvm+cancel_existing_cryptlvm_pvm.yaml:3]: In this case ppc64le tests require `textmode` desktop in order to work.
I talked with the team about the possibility to swap to `gnome` desktop but it seems that Gnome desktop requires at least 4GB of RAM and the defined machine `spvm` use LPAR which doesn't allow to increase the RAM size by vars.
They suggest me to swap machines to `hmc`, giving a short test it doesn't work easily so I ended up at the only way I found to make all architectures to work. Maybe I can touch the default YAML files but it seems to be pretty breaker at multiple points to me.

**Comment by okynos** [comment]
Good to know, I will try this in the next PR

**Pull request merged**